.github/workflows/ci.yml

```diff
@@ -17,9 +17,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.6, 3.7, 3.8, 3.9, "pypy-3.7"]
         os: [ubuntu-latest, macos-latest]
         platform: [x64]
+        exclude:
+          - platform: macos-latest
```

Comment:
did you mean "os" instead of "platform"? I see the macos pypy3 build is still being run

Reply:
Oops :o